src/config.rs

```diff
@@ -97,6 +122,21 @@ impl Config {
                 .value_name("percentile")
                 .help("Generate a report of requests over a given latency"))
 
+            .arg(Arg::new("http_method")
+                .short('m')
+                .long("method")
+                .value_name("http_method")
+                .possible_values(&["GET", "POST", "PUT", "DELETE"])
```

Comment:
Lets drop `DELETE` from this ... I know its for completeness, but

Reply:
@markpritchard 
I've removed `DELETE`. Also addressed a few clippy suggestions